### PR TITLE
fix: SLA parameteres in range

### LINKED ISSUE #9402: [Bug]: Market param update proposal enacted for out of range values
### Problem encountered

The following market parameters when updated via proposal with a value outside the valid range defined in the AC are being enacted

```
market.liquidity.priceRange with updated value of 101 - 0044-LIME-084
market.liquidity.slaCompetitionFactor with updated value of "1.1" - 0044-LIME-085
market.liquidity.performanceHysteresisEpochs with updated value of 367 - 0044-LIME-086
```

### Observed behaviour

param change proposals are enacted when value is out of range

### Expected behaviour

param change proposals are rejected when value is out of range

### Steps to reproduce

```bash
Run jenkins job from branch `0044-LIME-083` for the following tests

- test_boundary_update_price_range
- test_boundary_update_sla_competition_factor
- test_boundary_update_epoch_hysteresis
```

### Software version

dev branch

### Failing test

mentioned in: Steps to reproduce

### Jenkins run

https://jenkins.vega.rocks/job/common/job/system-tests-wrapper/8846/

### Configuration used

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by karlem**
Close #9402

### CODE REVIEW

**Review by jeremyletang — DISMISSED** [review]

**Comment by wwestgarth** [comment]
Isn't the issue that we're missing validating the SLA parameters in the update market command validation in:
```
checkUpdateMarketChanges()
```

**Comment by jeremyletang** [comment]
> Isn't the issue that we're missing validating the SLA parameters in the update market command validation in:
> 
> ```
> checkUpdateMarketChanges()
> ```
> 
> we shouldn't have made it all the way to the governance engine?

that's correct. I actually approved a little bit too quick, the validation should be in the commands package.

File: core/governance/market.go [hunk 1]
```diff
@@ -655,17 +655,18 @@ func validateLPSLAParams(slaParams *types.LiquiditySLAParams) (types.ProposalErr
 	if slaParams == nil {
 		return types.ProposalErrorMissingSLAParams, fmt.Errorf("liquidity provision SLA must be provided")
 	}
-	if slaParams.PriceRange.LessThanOrEqual(num.DecimalZero()) {
+	if slaParams.PriceRange.LessThan(num.DecimalOne()) || slaParams.PriceRange.GreaterThan(num.DecimalFromFloat(100)) {
```
Comment (wwestgarth) [inline core/governance/market.go:658]: This doesn't match the command validation that doesn't allow a price range of 0
```
lppr.IsNegative() || lppr.IsZero() {
    errs.AddForProperty(fmt.Sprintf("%s.price_range", parent), ErrMustBePositive)
}
```

Comment (karlem) [reply core/governance/market.go:658]: because it can't be 0.
``
market.liquidity.priceRange valid values: >0, <=100 
``

**Review by wwestgarth — DISMISSED** [review]

**Review by jeremyletang — APPROVED** [review]

**Pull request merged**